features/filesystem/littlefs/littlefs/lfs.h

```diff
@@ -259,9 +259,9 @@ typedef struct lfs_superblock {
 } lfs_superblock_t;
 
 typedef struct lfs_free {
-    lfs_block_t begin;
-    lfs_block_t size;
     lfs_block_t off;
+    lfs_block_t size;
+    lfs_block_t i;
```

Comment:
Can we have better name for `i`?  Other variables pretty much explain the purpose, except i.

Reply:
Anything but `i` :-) My guess without looking at what it does - just a counter so user does not need to initialize another variable `for (int i = 0...)` 😄

Reply:
Well, that's exactly what it is... a counter, it's state just lasts outside of a function

Reply:
Then name it counter_for_something (replace somthing what it counts, what value it holds)

Reply:
Counter / index ?

Reply:
Nah, I like count_for_what_it_counts

Reply:
I joke, sure I can change it to index once I'm somewhere I can push to github

Reply:
Is updated